src/main/java/core/data/medImage/MedImageAttributeMockup.java

```diff
@@ -0,0 +1,18 @@
+package core.data.medImage;
+
+/**
+ * Class to store information about {@link MedImageAttribute} signature, without actual value
+ */
+public class MedImageAttributeMockup {
```

Comment:
We can make value an optional field for MedImageAttribute, so we don't need a separate class when we just dont want to hold any value.